digitalsynth/src/components/css/Footer.css

```diff
@@ -1,10 +1,11 @@
 .FooterNav {
-  width: 100vw;
+  width: 89vw;
```

Comment:
This should be handled directly by its parent element's flexbox. You have some classes competing in App.css and here, one should be enough